java/src/main/java/com/logicalclocks/hsfs/metadata/HopsworksHttpClient.java

```diff
@@ -19,10 +19,34 @@
 import com.logicalclocks.hsfs.FeatureStoreException;
 import org.apache.http.HttpRequest;
 import org.apache.http.client.ResponseHandler;
+import org.slf4j.Logger;
 
+import java.io.FileInputStream;
 import java.io.IOException;
 
 public interface HopsworksHttpClient {
+  static final Logger LOGGER = null;
+
   <T> T handleRequest(HttpRequest request, ResponseHandler<T> responseHandler)
       throws IOException, FeatureStoreException;
+
+  String getTrustStorePath();
+
+  String getKeyStorePath();
+
+  String getCertKey();
+
+  static String readCertKey(String materialPwd) {
+    try (FileInputStream fis = new FileInputStream(materialPwd)) {
+      StringBuilder sb = new StringBuilder();
+      int content;
+      while ((content = fis.read()) != -1) {
+        sb.append((char) content);
+      }
+      return sb.toString();
+    } catch (IOException ex) {
+      LOGGER.warn("Failed to get cert password.", ex);
+    }
+    return null;
+  }
```

Comment:
Use the commons-io library, there is a FileUtils.readFileToString method which makes reading a String from a file a one-liner.

Reply:
replaced as you said, dunno where I got this monster from